src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -12,6 +12,7 @@ internal enum ConversionKind : byte
         Identity,
         ImplicitNumeric,
         ImplicitEnumeration,
+        ImplicitTuple,
```

Comment:
I would expect us to have ExplicitTuple conversion when there is an explicit conversion from elements to the corresponding target types.